test/e2e/common_suite_test.go

```diff
@@ -292,7 +344,7 @@ func getSuccessfulAndErroredPods(ctx context.Context, t *testing.T, client klien
 func doTestCreateSimplePod(t *testing.T, assert CloudAssert) {
 	namespace := envconf.RandomName("default", 7)
 	pod := newNginxPod(namespace)
-	newTestCase(t, "SimplePeerPod", assert, "PodVM is created").withPod(pod).run()
+	newTestCase(t, "SimplePeerPod", assert, "PodVM is created").withPod(pod).withCustomPodState(v1.PodRunning).run()
```

Comment:
I'd make the `newTestCase` function add a default pod state `v1.PodRunning` to the test case so you don't have to use `withCustomPodState(v1.PodRunning)` when creating every test case.